login1/dbus.go

```diff
@@ -155,6 +156,72 @@ func userFromInterfaces(user []interface{}) (*User, error) {
 	return &ret, nil
 }
 
+//GetActiveSession may be used to get the session object path for the current active session
+func (c *Conn) GetActiveSession() (dbus.ObjectPath, error) {
+	var activeSessionPath dbus.ObjectPath
+	var seat0Path dbus.ObjectPath
+	err := c.object.Call(dbusInterface+".GetSeat", 0, "seat0").Store(&seat0Path)
+
+	if err != nil {
+		return activeSessionPath, err
```

Comment:
Here and everywhere else, please `return nil, err` on error (or `""`, or whatever variation works).